x-pack/plugins/lens/public/indexpattern_datasource/types.ts

```diff
@@ -7,7 +7,7 @@
 
 import type { IndexPatternAggRestrictions } from '@kbn/data-plugin/public';
 import type { FieldSpec } from '@kbn/data-plugin/common';
-import type { FieldFormatParams } from '@kbn/field-formats-plugin/common';
+import type { FieldFormatMap } from '@kbn/data-views-plugin/common';
```

Comment:
I'm not sure why `FieldFormat` and `FieldFormatParams` are exported from the `field-formats` plugin,

but `FieldFormatMap` is exported from the `data-views` plugin. :confused:

Reply:
FieldFormatMap is how data views saves field formatters